src/clients/__tests__/KeyknoxClient.test.ts

```diff
@@ -1,18 +1,25 @@
-import { VirgilCrypto, VirgilAccessTokenSigner } from 'virgil-crypto';
+import { Buffer as NodeBuffer } from 'buffer';
+import { expect } from 'chai';
+
+import { initCrypto, VirgilCrypto, VirgilAccessTokenSigner } from 'virgil-crypto';
 import { Jwt, JwtGenerator } from 'virgil-sdk';
-import * as uuid from 'uuid/v4';
+import uuid from 'uuid/v4';
 
 import KeyknoxClient from '../KeyknoxClient';
 
 describe('KeyknoxClient', () => {
   let client: KeyknoxClient;
   let jwt: Jwt;
 
-  beforeEach(() => {
+  beforeEach(async () => {
+    await initCrypto();
```

Comment:
This can be in the `before()` method so that it only runs once per the entire suite